src/dao/pages/DAOProposals/CreateDAOProposal.tsx

```diff
@@ -30,16 +30,22 @@ import {
   useCreateMultiSigDAOUpgradeProposal,
   useDAOs,
   useCreateGOVTokenAssociateProposal,
+  useContract,
 } from "@dao/hooks";
 import { useHandleTransactionSuccess, useAccountTokenBalances } from "@dex/hooks";
 import { isNil } from "ramda";
 import { TransactionResponse } from "@hashgraph/sdk";
 import { getLastPathInRoute } from "@dex/utils";
 import { getDAOType, getPreviousMemberAddress } from "../utils";
-import { solidityAddressToAccountIdString } from "@shared/utils";
+import { isNFT, solidityAddressToAccountIdString } from "@shared/utils";
 
 export function CreateDAOProposal() {
   const { accountId: daoAccountId = "" } = useParams();
+  const daoAccountIdQueryResults = useContract(daoAccountId);
```

Comment:
If `accountId`/`daoAccountId` is an EVM address and no longer an Account Id, appropriately rename the variable to explicitly highlight this change.